chart/templates/externalpostgresql-configmap.yaml

```diff
@@ -0,0 +1,16 @@
+{{- if and .Values.externalPostgresql.sslEnabled .Values.externalPostgresql.sslCA }}
```

Comment:
That's a good idea, use another configmap. The problem is that it breaks the organization of variables.
I think that we have 2 options:
- Keep that file, move all postgres related variables of all the components to that file & mount the configmap in all components that require postgres
- Delete that file and put the certificate on each configmap of each component.

Both options implies to modify the volumes mounted on the deployments to specify the key to be mounted, as we already do with secrets